contents/teams/feature-flags/objectives.mdx

```diff
@@ -1,54 +1,86 @@
-## Q3 2025 Objectives
+## Q4 2025 Objectives
 
-### Objective 1: Behavioral Cohort Targeting
+### Objective 1: Awesome documentation <TeamMember name="Dylan Martin" photo />
+Motivation: Present feature flags as a mature and sophisticated product through excellent documentation, following the experiments team's successful approach.
 
-Motivation: Enable dynamic behavioral targeting across PostHog – let customers target users based on what they've done (or haven't done) recently. In other words, enable customers to target feature flags on [behavioral cohorts](/docs/data/cohorts).
+What we will ship:
+* Partner with the [content team](/teams/content) for comprehensive refresh
+* Clear structure covering all SDK types and evaluation methods  
+* Real-world scenario walkthroughs
+* AI-related content – examples of how to use flags with AI features
+* Easy navigation from app to relevant docs
 
-This is an objective shared across multiple teams ([Messaging](/teams/messaging), [Feature Flags](/teams/feature-flags), [Ingestion](/teams/ingestion), and [Surveys](/teams/surveys)) who all need behavioral targeting to enhance their products. We'll build a platform that integrates with our ingestion pipeline to pre-process event data for targeting, then integrate this service into existing product targeting functionality.
+### Objective 2: Make local evaluation the default <TeamMember name="Dylan Martin" photo /> <TeamMember name="Phil Haack" photo />
+Motivation: Enable faster, more reliable flag evaluations by making local evaluation the default for server-side SDKs, while also improving the durability of the `/local_evaluation` endpoint itself.
 
-We'll deliver this in 3 stages with Stage 1 well-defined and Stages 2-3 evolving based on how Stage 1 goes. Note that stages can overlap – we may begin customer partnerships and product integrations as soon as the POC works well enough, potentially before Stage 2 is completed.
+What we will ship:
+* **HyperCache + local eval completion** – Finish implementation of `/local_evaluation` powered by HyperCache <TeamMember name="Phil Haack" photo />
+* **Properties solution** – Solve the blocker preventing wider local eval adoption: make it easier for users to provide properties needed for flag evaluation <TeamMember name="Dylan Martin" photo /> <TeamMember name="Phil Haack" photo />
+* **Better documentation & guides** – Create how-to guides for local evaluation setup <TeamMember name="Dylan Martin" photo /> <TeamMember name="Phil Haack" photo />
+* **Endpoint resilience** – Make the `/local_evaluation` endpoint completely resilient to database outages <TeamMember name="Phil Haack" photo />
+* **ETag support** – Implement efficient caching for reduced bandwidth <TeamMember name="Phil Haack" photo />
 
-**Stage 1: Proof-of-concept**
+### Objective 3: UI/UX improvements (Shared goal for the whole team)
```

Comment:
this could be a bit more specific – the idea of this goal is I want to track UI & UX improvements on our product; both through burning down the existing backlog but _also_ through doing more user interviews.